spec/controllers/admin/users_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+RSpec.describe Admin::UsersController, type: :controller do
+  include SessionsHelper
+
+  let(:admin){FactoryBot.create :user, role: "admin"}
+  let!(:user_1){FactoryBot.create :user}
+  let!(:user_2){FactoryBot.create :user}
+  before do
+    log_in admin
+  end
+
+  describe "GET users#index" do
```

Comment:
describe, context và it là gì em

Reply:
describe dùng để nhóm các test case theo class, module, method hoặc theo action của controller
context dùng để nhóm các điều kiện của example.
it dùng để định nghĩa 1 test case.